content/docs/components/serving/overview.md

```diff
@@ -0,0 +1,49 @@
++++
+title = "Overview"
+description = "Model Serving Overview"
+weight = 1
++++
+
+Kubeflow provides two supported open source model serving systems that allow multi-framework model serving: KFServing and Seldon Core. You should choose the framework that best supports your model serving requirements. If you only serve  Tensorflow models using Tensorflow Serving is an option although both KFServing and Seldon support Tensorflow. A rough comparison between KFServing and Seldon Core is shown below:
+
+| Feature        | sub-feature    | KFServing | Seldon |
```

Comment:
Is the brand name "Seldon" or "Seldon Core"? In other docs, I've assumed "Seldon Core" as that seems to be what the Seldon website goes with. But it may be that there's a meaningful difference between "Seldon" and "Seldon Core", in which case we should make sure we use the names properly. Please would you update this page to be consistent.